### PR TITLE
fix proxy readonly deprecated hooks error in webpack 5

### PR DESCRIPTION
**Pull request by tanhauhau**
Fixes https://github.com/stephencookdev/speed-measure-webpack-plugin/issues/149

### CODE REVIEW

File: __tests__/setups/v5-deprecated-hooks/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "v5-deprecated-hooks",
+  "version": "0.0.1",
+  "description": "Test webpack v4 with a StatsPlugin set-up with SMP",
```
Comment (rmariuzzo) [inline __tests__/setups/v5-deprecated-hooks/package.json:4]:
```suggestion
  "description": "Test webpack v5 with a StatsPlugin set-up with SMP",
```

**Review by rmariuzzo — CHANGES_REQUESTED** [review]
LGTM, one small change.

**Review by simPod — APPROVED** [review]

**Comment by brybrophy** [comment]
@rmariuzzo @simPod Can we get this merged, please? This is blocking my team from being able to upgrade to webpack 5.

**Comment by stephencookdev** [comment]
Nice, this looks great — awesome stuff @tanhauhau! I'll merge this into the upcoming 1.4.0 release :)

**Pull request merged**